DETAILED_INSTALL.md

```diff
@@ -2,251 +2,79 @@
 
 ### Docker
 
-#### Using pre-built images
+#### Building the image
 
-You can pull & run the latest pre-built images from our [DockerHub](https://hub.docker.com/r/instadeepct/mava) by specifying the docker image and example/file you want to run.
-
-For example, this will pull the latest mava jax core image and run the `examples/debugging/simple_spread/feedforward/decentralised/run_ippo.py` example:
-
-```
-docker run --gpus all -it --rm  -v $(pwd):/home/app/mava -w /home/app/mava instadeepct/mava:jax-core-latest python examples/debugging/simple_spread/feedforward/decentralised/run_ippo.py --base_dir /home/app/mava/logs/
-```
-
-* For windows, replace `$(pwd)` with `$(curdir)`.
-
-* You can replace the example with your custom python file.
-
-#### Building the image yourself
-
-1. Build the correct docker image using the `make` command:
+1. Build the docker image using the `make` command:
```

Comment:
```suggestion
1. Build the docker image using the `make build` command:
```

Reply:
I think we can keep it as is? Since we say `make build` a few steps down?